www-client/seamonkey/seamonkey-2.53.15-r1.ebuild

```diff
@@ -0,0 +1,539 @@
+	x86? ( >=dev-lang/yasm-1.1 )
+"
+COMMON_DEPEND="
+	app-arch/bzip2
+	>=app-text/hunspell-1.5.4:=
+	dev-libs/atk
+	>=dev-libs/glib-2.26:2
+	>=dev-libs/libffi-3.0.10:=
+	>=dev-libs/nspr-4.23
+	>=dev-libs/nss-3.47.1
+	media-libs/fontconfig
+	>=media-libs/freetype-2.4.10
+	>=media-libs/mesa-10.2:=
+	>=sys-libs/zlib-1.2.3
+	>=x11-libs/cairo-1.10[X]
+	x11-libs/gdk-pixbuf
+	x11-libs/gtk+:3[X]
+	x11-libs/libX11
+	x11-libs/libXcomposite
+	x11-libs/libXdamage
+	x11-libs/libXext
+	x11-libs/libXfixes
+	x11-libs/libXrender
+	x11-libs/libXt
+	x11-libs/libxcb:=
+	>=x11-libs/pango-1.22.0
+	x11-libs/pixman
+	media-video/ffmpeg
+	virtual/freedesktop-icon-theme
+	dbus? (
+		>=dev-libs/dbus-glib-0.72
+		>=sys-apps/dbus-0.60
+	)
+	jack? ( virtual/jack )
+	kernel_linux? ( !pulseaudio? ( media-libs/alsa-lib ) )
+	pulseaudio? ( || (
+		media-sound/pulseaudio
+		>=media-sound/apulse-0.1.9
+	) )
+	startup-notification? ( >=x11-libs/startup-notification-0.8 )
+	system-av1? (
+		>=media-libs/dav1d-0.3.0:=
+		>=media-libs/libaom-1.0.0:=
+	)
+	system-harfbuzz? (
+		>=media-gfx/graphite2-1.3.9-r1
+		>=media-libs/harfbuzz-1.3.3:0=
+	)
+	system-icu? ( >=dev-libs/icu-59.1:= )
+	system-jpeg? ( >=media-libs/libjpeg-turbo-1.2.1 )
+	system-libevent? ( >=dev-libs/libevent-2.0:0= )
+	system-libvpx? ( >=media-libs/libvpx-1.8.0:0=[postproc] )
+	system-png? ( >=media-libs/libpng-1.6.31:0=[apng] )
+	system-sqlite? ( >=dev-db/sqlite-3.38.2:3[secure-delete] )
+	wifi? (
+		kernel_linux? (
+			>=dev-libs/dbus-glib-0.72
+			net-misc/networkmanager
+			>=sys-apps/dbus-0.60
+		)
+	)
+"
+RDEPEND="${COMMON_DEPEND}
+	selinux? ( sec-policy/selinux-mozilla )
+"
+DEPEND="${COMMON_DEPEND}
+	x11-base/xorg-proto
+	amd64? ( virtual/opengl )
+	x86? ( virtual/opengl )
+"
+
+# allow GMP_PLUGIN_LIST to be set in an eclass or
+# overridden in the enviromnent (advanced hackers only)
+[[ -z ${GMP_PLUGIN_LIST} ]] && GMP_PLUGIN_LIST=( gmp-gmpopenh264 gmp-widevinecdm )
+
+BUILD_OBJ_DIR="${S}/seamonk"
+
+pkg_setup() {
+	if [[ ${PV} == *_beta* ]] || [[ ${PV} == *_pre* ]] ; then
+		ewarn
+		ewarn "You're using an unofficial release of ${PN}. Don't file any bug in"
+		ewarn "Gentoo's Bugtracker against this package in case it breaks for you."
+		ewarn "Those belong to upstream: https://bugzilla.mozilla.org"
+	fi
+
+	moz_pkgsetup
+}
+
+pkg_pretend() {
+	# Ensure we have enough disk space to compile
+	if use lto || use test ; then
+		CHECKREQS_DISK_BUILD="16G"
+	else
+		CHECKREQS_DISK_BUILD="12G"
+	fi
+	check-reqs_pkg_setup
+}
+
+spkg_setup() {
+	# Ensure we have enough disk space to compile
+	if use lto || use test ; then
+		CHECKREQS_DISK_BUILD="16G"
+	else
+		CHECKREQS_DISK_BUILD="12G"
+	fi
+	check-reqs_pkg_setup
+}
+
+src_unpack() {
+	local l10n_sources="${P}.source-l10n.tar.xz"
+	unpack ${A/ ${l10n_sources}}
+
+	mkdir "${S}/${P}-l10n" || die
+	cd "${S}/${P}-l10n" || die
+	unpack ${l10n_sources}
+}
+
+src_prepare() {
+	# Apply our patches
+	eapply "${WORKDIR}"/gentoo-${PN}-patches-${PVR}/${PN}
```

Comment:
I'd discourage using PVR here if you can, and instead introduce your own _pN to the ebuild version.

The reason being that a revision might be made for other reasons (like fixing a dependency) where it doesn't make sense.

But I won't block merging this now, could you handle it in a followup?

Many thanks for the hard work on seamonkey.

Reply:
Yeah, didn't think about that. No problem. Will fix it in a follow-up.